tests/binary_agreement.rs

```diff
@@ -108,3 +221,9 @@ fn test_binary_agreement_first_silent() {
     let new_adversary = |_: usize, _: usize| SilentAdversary::new(MessageScheduler::First);
     test_binary_agreement_different_sizes(new_adversary);
 }
+
+#[test]
+fn test_binary_agreement_reordering() {
+    let new_adversary = |_: usize, _: usize| ReorderingAdversary::default();
+    test_binary_agreement_different_sizes(new_adversary);
```

Comment:
Does it really make sense for this particular test to run with `InputConfig`s other than `Split`?